compiler/z/codegen/OMRCodeGenerator.cpp

```diff
@@ -457,6 +457,15 @@ TR_S390ProcessorInfo::checkZ13()
    }
 
 ////////////////////////////////////////////////////////////////////////////////
+// TR_S90ProcessorInfo::checkZ14: returns true if it's z14, otherwise false.
+////////////////////////////////////////////////////////////////////////////////
+bool
+TR_S390ProcessorInfo::checkZ14()
+   {
+   return TR::Compiler->target.cpu.getS390SupportsZ14();
+   }
+
+////////////////////////////////////////////////////////////////////////////////
 // TR_S90ProcessorInfo::checkZNext: returns true if it's zNext, otherwise false.
```

Comment:
Fixed in cb358ed. I also fixed a few other occurrences while in this area of code.